components/collapsible-panel/collapsible-panel-group.js

```diff
@@ -0,0 +1,71 @@
+import { css, html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+
+/**
+ * A component that renders collapsible panels
```

Comment:
I think "A component that renders a container and layout for collapsible panels" would be a more accurate description.

Reply:
Oh yeah, words...I forgot about updating this. Yeah I like your suggestion.